command/base/command.go

```diff
@@ -130,6 +140,9 @@ func (c *Command) Parse(args []string) error {
 
 // Help returns the help for this flagSet.
 func (c *Command) Help() string {
+	if c.flagSet == nil {
+		return ""
+	}
```

Comment:
I wasn't sure if there was a better solution than this for cases where Help() gets called before anything is initialized, like with the kv command and the kv tests that check for no tabs.